tvsaver/saver2v2/save_file_test.go

```diff
@@ -218,8 +242,14 @@ func TestSaver2_2FileWrapsCopyrightMultiLine(t *testing.T) {
 	f := &spdx.File2_2{
 		FileName:           "/tmp/whatever.txt",
 		FileSPDXIdentifier: spdx.ElementID("File123"),
-		FileChecksumSHA1:   "85ed0817af83a24ad8da68c2b5094de69833983c",
-		LicenseConcluded:   "Apache-2.0",
+		FileChecksums: map[spdx.ChecksumAlgorithm]spdx.Checksum{
+			spdx.SHA1: spdx.Checksum{
+				Algorithm: spdx.SHA1,
+				Value:     "85ed0817af83a24ad8da68c2b5094de69833983c",
+			},
+		},
+
```

Comment:
Redundant NL again. Is this due to the IDE you're using?
We have similar NLs all over the commit.

Reply:
@specter25 , you've done a great job. Appreciate your persistence.